src/kakarot/accounts/uninitialized_account.cairo

```diff
@@ -0,0 +1,43 @@
+// SPDX-License-Identifier: MIT
+
+%lang starknet
+
+// Starkware dependencies
+from openzeppelin.access.ownable.library import Ownable
+from starkware.cairo.common.cairo_builtins import HashBuiltin, BitwiseBuiltin, SignatureBuiltin
+from starkware.cairo.common.uint256 import Uint256
+
+// Local dependencies
+from kakarot.accounts.library import GenericAccount, Account_kakarot_address, Account_evm_address
```

Comment:
don't import things from kakarot otherwise the class of this account depends on the kakarot library, which might change